.github/workflows/backport.yml

```diff
@@ -103,6 +103,12 @@ jobs:
           git config user.name "${{ env.AUTHOR_NAME }}"
           git fetch
           git cherry-pick -x --mainline 1 ${{ env.COMMIT }}
+          make fmt vet generate apidoc wordlist-ordered
+          if ! git diff --exit-code --quiet
```

Comment:
The option `--quiet` implies `--exit-code` so I don't think it's needed, but it could be nice to have since  it helps to make it clear what we're expecting from this exit code